<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Download Sito Web</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>URL da scaricare:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="url"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Ignora file con estensioni:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ignoreext"/>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Timeout (secondi):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="timeout">
           <property name="maximum">
            <number>300</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Opzioni</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="usevdb">
            <property name="text">
             <string>Usa VdB per riconoscere il testo italiano</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="recursive">
            <property name="text">
             <string>Ricorsivo</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="keeplinkscontent">
            <property name="text">
             <string>Mantieni didascalia dei link</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="download">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(81, 244, 0);</string>
         </property>
         <property name="text">
          <string>Scarica</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Ricerca Repubblica.it</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Da</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Giorno</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="dagiorno">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>31</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Mese</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="damese">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>12</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Anno</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="daanno">
            <property name="minimum">
             <number>2000</number>
            </property>
            <property name="maximum">
             <number>2100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>A</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Giorno</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="agiorno">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>31</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Mese</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="amese">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>12</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Anno</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="aanno">
            <property name="minimum">
             <number>2000</number>
            </property>
            <property name="maximum">
             <number>2100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Query:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="repquery"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="searchrep">
         <property name="text">
          <string>Scarica da www.repubblica.it</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <property name="accessibleName">
       <string>Twitter</string>
      </property>
      <attribute name="title">
       <string>Twitter</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Nome utente da scaricare:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="twittername"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QRadioButton" name="astxt">
           <property name="text">
            <string>Scarica co&amp;me TXT</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ascsv">
           <property name="text">
            <string>Scarica come CS&amp;V</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Consumer Key:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Consumer Secret:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Access Key:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Access Secret:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="consumer_key"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="consumer_secret"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="access_key"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="access_secret"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="dotwitter">
         <property name="text">
          <string>Scarica feed twitter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="folder"/>
     </item>
     <item>
      <widget class="QPushButton" name="choosefolder">
       <property name="text">
        <string>Scegli cartella</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="resultsgrp">
     <property name="title">
      <string>Sto scaricando:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QListWidget" name="results"/>
      </item>
      <item>
       <widget class="QPushButton" name="stopall">
        <property name="text">
         <string>Interrompi tutte le operazioni</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
